ci/jjb/scripts/pulp-install-f28.sh

```diff
@@ -0,0 +1,12 @@
+setenforce permissive
```

Comment:
@ragabala, perhaps just skip just the failing test due to selinux issues can be another solution.

Reply:
@kersommoura: The issue is not in tests .. but in the installation of pulp. The pulp-ci fails to install pulp as squid fails to start in the lazy role if SELinux is enabled in F28.